BEIMA.Backend.FT/DeviceFT.cs

```diff
@@ -186,8 +186,7 @@ public async Task DevicesInDatabase_GetDeviceList_ReturnsDeviceList()
                 Assert.That(device.LastModified?.User, Is.EqualTo("Anonymous"));
             }
         }
-
-        [Test]
+        
```

Comment:
```suggestion
        [Test]
```
Left out test tag on merge, woops.